From d3d99706ec0ecdcff75bf16f2d37faa6fd154316 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <home@chrisaw.com>
Date: Sun, 31 Mar 2024 12:29:37 +0100
Subject: [PATCH 1/1] Remove memtag_globals references

---
 third_party/boringssl/Android.bp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/third_party/boringssl/Android.bp b/third_party/boringssl/Android.bp
index 499d978b..b79adb1e 100644
--- a/third_party/boringssl/Android.bp
+++ b/third_party/boringssl/Android.bp
@@ -116,7 +116,6 @@ cc_object {
         // to help prevent
         // merge conflicts.
         fuzzer: false,
-        memtag_globals: false,
     },
     target: {
         android: {
@@ -162,7 +161,6 @@ cc_object {
         address: false,
         hwaddress: false,
         fuzzer: false,
-        memtag_globals: false,
     },
     target: {
         android: {
-- 
2.44.0
