From f4afe66a02c3804f3cc8c62c225f5382125c655f Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <home@chrisaw.com>
Date: Mon, 6 May 2024 15:59:11 +0100
Subject: [PATCH 2/4] Disable enforcing unauthenticated adb

---
 target/board/gsi_system_ext.prop | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/target/board/gsi_system_ext.prop b/target/board/gsi_system_ext.prop
index 780aadc..217bd01 100644
--- a/target/board/gsi_system_ext.prop
+++ b/target/board/gsi_system_ext.prop
@@ -1,9 +1,6 @@
 # GSI always generate dex pre-opt in system image
 ro.cp_system_other_odex=0
 
-# GSI always disables adb authentication
-ro.adb.secure=0
-
 # GSI disables non-AOSP nnapi extensions on product partition
 ro.nnapi.extensions.deny_on_product=true
 
-- 
2.44.0
