From 5b213010c65c8fed617206f0af935df63ea5eec1 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <home@chrisaw.com>
Date: Sat, 27 Jan 2024 23:00:01 +0000
Subject: [PATCH 3/3] set ccache directory to build dir

---
 envsetup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/envsetup.sh b/envsetup.sh
index 23f1e1b..25c0450 100644
--- a/envsetup.sh
+++ b/envsetup.sh
@@ -2137,7 +2137,7 @@ if [ -z ${CCACHE_EXEC} ]; then
         export USE_CCACHE=1
         export CCACHE_EXEC="$ccache_path"
         if [ -z ${CCACHE_DIR} ]; then
-            export CCACHE_DIR=${HOME}/.ccache
+            export CCACHE_DIR=$(pwd)/.ccache
         fi
         $ccache_path -o compression=true
         echo -e "ccache enabled and CCACHE_EXEC has been set to : $ccache_path"
-- 
2.42.0
