From 3da5b943fa3813034532b2da6ed78359b0f70405 Mon Sep 17 00:00:00 2001
From: ahnet-69 <syedmusicrealise@gmail.com>
Date: Sun, 24 Dec 2023 11:49:10 +0500
Subject: [PATCH] Remove definations of ro.adb.secure

---
 base.mk    |  1 -
 1 files changed, 1 deletion(-)

diff --git a/base.mk b/base.mk
index 83bb3f7..91d523c 100644
--- a/base.mk
+++ b/base.mk
@@ -33,6 +33,5 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
 	ro.build.version.all_codenames=$(PLATFORM_VERSION_ALL_CODENAMES) \
 	ro.build.version.release=$(PLATFORM_VERSION) \
 	ro.build.version.security_patch=$(PLATFORM_SECURITY_PATCH) \
-	ro.adb.secure=0 \
 	ro.logd.auditd=true \
 	ro.logd.kernel=true \
--
2.34.1

