From 8d9f5f78c7629aa9a416c706a0380fd3e1ffa03e Mon Sep 17 00:00:00 2001
From: chrisaw <home@chrisaw.com>
Date: Thu, 28 Dec 2023 22:54:12 +0000
Subject: [PATCH] Fix gapps path

---
 gapps.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gapps.mk b/gapps.mk
index 15e12a9..a9390fb 100644
--- a/gapps.mk
+++ b/gapps.mk
@@ -44,7 +44,7 @@ endif
 #Pixel Experience gapps
 ifneq ($(wildcard vendor/gapps),)
 IS_PHONE := true
-$(call inherit-product,vendor/gapps/config.mk)
+$(call inherit-product,vendor/gapps/common/common-vendor.mk)
 PRODUCT_PACKAGES += \
        phh-overrides \
           com.simplemobiletools.gallery.pro \
-- 
2.34.1
