From f1c3ec0891b3f786df5efb53a712c17e9ac14744 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <home@chrisaw.com>
Date: Tue, 6 Feb 2024 18:45:02 +0000
Subject: [PATCH 8/11] Disable face unlock on a64 due to missing libs

---
 tdgsi_a64_ab/BoardConfig.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tdgsi_a64_ab/BoardConfig.mk b/tdgsi_a64_ab/BoardConfig.mk
index f131e88..551bb8f 100644
--- a/tdgsi_a64_ab/BoardConfig.mk
+++ b/tdgsi_a64_ab/BoardConfig.mk
@@ -8,3 +8,5 @@ ifeq ($(BOARD_SYSTEMIMAGE_PARTITION_RESERVED_SIZE),)
 BOARD_SYSTEMIMAGE_PARTITION_SIZE := 1572864000
 endif
 TARGET_USES_64_BIT_BINDER := true
+
+BUILD_BROKEN_MISSING_REQUIRED_MODULES := true
-- 
2.42.0
