From 7f2e3b216182db76ee59c9816d90c783288b5210 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <home@chrisaw.com>
Date: Mon, 22 Jan 2024 16:06:43 +0000
Subject: [PATCH 3/7] Disable generated kernel headers

---
 build/soong/Android.bp | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/build/soong/Android.bp b/build/soong/Android.bp
index 700ac69..448cae0 100644
--- a/build/soong/Android.bp
+++ b/build/soong/Android.bp
@@ -20,9 +20,6 @@ bootstrap_go_package {
 voltage_generator {
     name: "generated_kernel_includes",
 
-    // The headers make command
-    cmd: "$(PATH_OVERRIDE_SOONG) $(KERNEL_MAKE_CMD) $(KERNEL_MAKE_FLAGS) -C $(TARGET_KERNEL_SOURCE) O=$(KERNEL_BUILD_OUT_PREFIX)$(genDir) ARCH=$(KERNEL_ARCH) $(KERNEL_CROSS_COMPILE) headers_install",
-
     // Directories that can be imported by a cc_* module generated_headers property
     export_include_dirs: [
         "usr/audio/include/uapi",
@@ -31,15 +28,6 @@ voltage_generator {
         "usr/include/audio/include/uapi",
         "usr/techpack/audio/include",
     ],
-
-    // Sources for dependency tracking
-    dep_root: "$(TARGET_KERNEL_SOURCE)",
-    dep_files: [
-        "Makefile",
-        "include/**/*",
-        "arch/$(KERNEL_ARCH)/include/**/*",
-        "techpack/audio/include/**/*",
-    ],
 }
 
 cc_defaults {
-- 
2.42.0
