From f5edaa7ed2740249a7ad52b16de1edc3442c6543 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <home@chrisaw.com>
Date: Tue, 13 Feb 2024 09:27:42 +0000
Subject: [PATCH 5/6] Add GrapheneOS Apps

---
 config/common.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/config/common.mk b/config/common.mk
index 5457bbc..1a8956a 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -179,6 +179,10 @@ PRODUCT_PACKAGES += \
 PRODUCT_PACKAGES += \
     BtHelper
 
+# GrapheneOS Apps
+PRODUCT_PACKAGES += \
+    Apps
+
 # RRO
 include vendor/voltage/config/rro_overlays.mk
 
-- 
2.42.0
