From 382e0aa115e602ffe525744415a2e03823da5e38 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <home@chrisaw.com>
Date: Wed, 14 Feb 2024 22:44:39 +0000
Subject: [PATCH 6/8] Add privileged extensions for aurora and fdroid

---
 config/common.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/config/common.mk b/config/common.mk
index 1a8956a..e7d936c 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -175,6 +175,11 @@ PRODUCT_PACKAGES += \
 #    AndroidSnowPaintDropTheme \
 #    AndroidEspressoTheme
 
+# Aurora Store & F-Droid privileged extensions
+PRODUCT_PACKAGES += \
+    AuroraStorePrivilegedExtension \
+    F-DroidPrivilegedExtension
+
 # BtHelper
 PRODUCT_PACKAGES += \
     BtHelper
-- 
2.42.0
