From 3a7ded7266244890d06cda004ab16add6a5990b2 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <home@chrisaw.com>
Date: Sat, 17 Feb 2024 14:15:29 +0000
Subject: [PATCH] Add missing sdmcore_has_is_display_hw_available_func_defaults
 ref

---
 build/soong/Android.bp | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/build/soong/Android.bp b/build/soong/Android.bp
index 448cae0..18553a1 100644
--- a/build/soong/Android.bp
+++ b/build/soong/Android.bp
@@ -67,6 +67,15 @@ soong_config_module_type {
     properties: ["cppflags"],
 }
 
+sdmcore_has_is_display_hw_available {
+    name: "sdmcore_has_is_display_hw_available_func_defaults",
+    soong_config_variables: {
+        sdmcore_has_is_display_hw_available_func: {
+            cflags: ["-DSDMCORE_HAS_IS_DISPLAY_HW_AVAILABLE_FUNC"],
+        },
+    },
+}
+
 camera_override_format_from_reserved {
     name: "camera_override_format_from_reserved_defaults",
     soong_config_variables: {
-- 
2.42.0

