From 5ef5a8094058b19a31e48d7fad581d55020e9959 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <home@chrisaw.com>
Date: Mon, 19 Feb 2024 16:52:41 +0000
Subject: [PATCH 7/7] Switch LineageOS Aperture to GrapheneOS Camera

---
 config/common_full.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common_full.mk b/config/common_full.mk
index 4a7147d..daee5e5 100644
--- a/config/common_full.mk
+++ b/config/common_full.mk
@@ -5,5 +5,5 @@ PRODUCT_SIZE := full
 
 ifneq ($(PRODUCT_NO_CAMERA),true)
 PRODUCT_PACKAGES += \
-    Aperture
+    Camera
 endif
-- 
2.42.0
