From f45ac6eb189100d5a4be6910eb13e6fa6f4c95fc Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <home@chrisaw.com>
Date: Wed, 6 Mar 2024 20:06:56 +0000
Subject: [PATCH] Add Lawnchair launcher

---
 config/common.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/config/common.mk b/config/common.mk
index e7d936c..d86d9eb 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -188,6 +188,9 @@ PRODUCT_PACKAGES += \
 PRODUCT_PACKAGES += \
     Apps
 
+# Lawnchair Launcher
+$(call inherit-product-if-exists, vendor/lawnchair/lawnchair.mk)
+
 # RRO
 include vendor/voltage/config/rro_overlays.mk
 
-- 
2.43.1
